packages/components/src/components/popover/popover.jsx

```diff
@@ -60,7 +61,11 @@ const Popover = ({
             )}
             {(popover_ref || !relative_render) && (
                 <TinyPopover
-                    isOpen={is_open ?? ((is_hovered && message) || (is_bubble_hover_enabled && is_bubble_hovered))}
+                    isOpen={
+                        is_quickStrategy
+                            ? is_open ?? (is_hovered && message)
+                            : is_open ?? ((is_hovered && message) || (is_bubble_hover_enabled && is_bubble_hovered))
+                    }
```

Comment:
if this change works fine we don't need to add extra prop
```suggestion
                   isOpen={
                    is_bubble_hover_enabled
                        ? is_open ?? ((is_hovered && message)  || (is_bubble_hover_enabled && is_bubble_hovered))
                        : is_open ?? (is_hovered && message)
                    } 
```

Reply:
Changed, hope it is ok now